engine/common/synchronization/engine.go

```diff
@@ -12,7 +12,7 @@ import (
 
 	"github.com/onflow/flow-go/engine"
 	"github.com/onflow/flow-go/engine/common/fifoqueue"
-	"github.com/onflow/flow-go/model/events"
+	"github.com/onflow/flow-go/model/chainsync"
```

Comment:
what is this?

Reply:
Just removed it. It was something leftover from fixing merge conflicts.